tools/docker/builder/openwrt/toolchains/entrypoint.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/sh
+
+cmd="$@"
+
+# Check if the command argument is a service command
+if [ "${cmd}" = "show-launch-script" ] ; then
+    cat /usr/local/bin/openwrt-env
```

Comment:
I don't see any mention of `openwrt-env` script in your Dockerfile.
Did you mean `prplwrt-env`?
If yes, then please make it a part of docker image